lib/shared/bucketing-assembly-script/assembly/bucketing/segmentation.ts

```diff
@@ -34,7 +34,10 @@ export function _evaluateOperator(
         // Replace Array.some() logic
         for (let i = 0; i < operator.filters.length; i++) {
             const filter = operator.filters[i]
-            if (doesUserPassFilter(filter, audiences, user, clientCustomData)) {
+            if (filter.operatorClass !== null) {
+                return _evaluateOperator(filter.operatorClass as AudienceOperator, audiences, user, clientCustomData)
```

Comment:
is the type assertion here just to remove the "null"?

Reply:
yep